<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_2, #label, #label_11, #label_12 {
	color: rgba(255,255,255,0.9);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QWidget" name="container" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1281</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#container {
	background-image: url(:/img/img/login-bg.png);
}</string>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>28</x>
         <y>40</y>
         <width>1220</width>
         <height>641</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: none;</string>
       </property>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>471</width>
          <height>641</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: none;</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>200</y>
           <width>321</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Source Sans 3</family>
           <pointsize>36</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Create Account</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>280</y>
           <width>361</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Source Sans 3</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>UpArrowCursor</cursorShape>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>Efficient and Convenient Exam Experience Like Never Before</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="registerPushButton">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>370</y>
           <width>190</width>
           <height>50</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 2px solid rgba(255,255,255,0.9);
	color: rgba(255,255,255,0.9);	
	border-radius: 25px;
}

QPushButton:hover {
	background-color: rgba(255,255,255,0.9);
	color: #009E7F;
}</string>
         </property>
         <property name="text">
          <string>SIGN UP</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>0</y>
          <width>761</width>
          <height>641</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background: none;</string>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>170</y>
           <width>296</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Source Sans 3</family>
           <pointsize>36</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #009E7F;</string>
         </property>
         <property name="text">
          <string>Welcom Back!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>240</y>
           <width>481</width>
           <height>111</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QLineEdit" name="emailLineEdit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>461</width>
            <height>51</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgba(0,0,0,0.2);
border-radius: 4px;
padding-left: 10px;
</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Source Sans 3</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #000;</string>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>350</y>
           <width>481</width>
           <height>101</height>
          </rect>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Source Sans 3</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #000;</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>461</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgba(0,0,0,0.2);
border-radius: 4px;
padding-left: 10px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="loginPushButton">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>470</y>
           <width>190</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>190</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color: rgba(255,255,255,0.9);
	background-color: #009E7F;
	border-radius: 25px;
}

QPushButton:hover 	{
	border: 2px solid #009E7F;
	color: #009E7F;	
	background-color: rgba(255,255,255,0.9);
}

</string>
         </property>
         <property name="text">
          <string>SIGN IN</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="container_2" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1281</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#container_2 {
	background-image: url(:/img/img/login-bg.png);
}</string>
      </property>
      <widget class="QWidget" name="widget_11" native="true">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>40</y>
         <width>1220</width>
         <height>641</height>
        </rect>
       </property>
       <widget class="QWidget" name="widget_12" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>471</width>
          <height>641</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>200</y>
           <width>221</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Source Sans 3</family>
           <pointsize>36</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>One of Us?</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>280</y>
           <width>321</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Source Sans 3</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>UpArrowCursor</cursorShape>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>If you already have your own account, just log in.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="loginPushButton_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>370</y>
           <width>190</width>
           <height>50</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border: 2px solid rgba(255,255,255,0.9);
	color: rgba(255,255,255,0.9);	
	border-radius: 25px;
}

QPushButton:hover {
	background-color: rgba(255,255,255,0.9);
	color: #009E7F;
}</string>
         </property>
         <property name="text">
          <string>SIGN IN</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="widget_13" native="true">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>0</y>
          <width>761</width>
          <height>641</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>70</y>
           <width>341</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Source Sans 3</family>
           <pointsize>36</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #009E7F;</string>
         </property>
         <property name="text">
          <string>Create Account!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QWidget" name="widget_14" native="true">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>270</y>
           <width>481</width>
           <height>111</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QLineEdit" name="resgisterEmailLineEdit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>461</width>
            <height>51</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgba(0,0,0,0.2);
border-radius: 4px;
padding-left: 10px;
</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Source Sans 3</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #000;</string>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_16" native="true">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>-120</y>
            <width>481</width>
            <height>111</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Source Sans 3</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #000;</string>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_15" native="true">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>380</y>
           <width>481</width>
           <height>101</height>
          </rect>
         </property>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Source Sans 3</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #000;</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="resgisterPasswordLineEdit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>461</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgba(0,0,0,0.2);
border-radius: 4px;
padding-left: 10px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="registerPushButton_2">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>500</y>
           <width>190</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>190</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color: rgba(255,255,255,0.9);
	background-color: #009E7F;
	border-radius: 25px;
}

QPushButton:hover 	{
	border: 2px solid #009E7F;
	color: #009E7F;	
	background-color: rgba(255,255,255,0.9);
}

</string>
         </property>
         <property name="text">
          <string>SIGN UP</string>
         </property>
        </widget>
        <widget class="QWidget" name="widget_17" native="true">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>160</y>
           <width>481</width>
           <height>111</height>
          </rect>
         </property>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>75</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Source Sans 3</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="resgisterNameLineEdit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>461</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgba(0,0,0,0.2);
border-radius: 4px;
padding-left: 10px;
</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
